PackageInfo.g

```diff
@@ -18,7 +18,7 @@ PackageName := "AutoDoc",
 Subtitle := "Generate documentation from GAP source code",
 
 Version := Maximum( [
-  "2016.02.23", ## Sebas' version
+  "2016.02.28", ## Sebas' version
```

Comment:
Please also update this date, and then perhaps use `git rebase --ignore-date` to prevent your commit from looking like they are several days old, when in fact you just rewrote them? :-)